source/accessibility.html.md.erb

```diff
@@ -0,0 +1,67 @@
+---
+title: Accessibility statement for GOV.UK Prototype Kit training documentation
+last_reviewed_on: 2020-09-01
+review_in: 6 months
+hide_in_navigation: true
+---
+
+# Accessibility statement for GOV.UK Prototype Kit training documentation
+
+This accessibility statement applies to the GOV.UK Prototype Kit training documentation at [https://prototype-kit-training.cloudapps.digital/](https://prototype-kit-training.cloudapps.digital/).
+
+This website is run by the GOV.UK Design System team at the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
++ change colours, contrast levels and fonts
++ zoom in up to 300% without the text spilling off the screen
++ navigate most of the website using just a keyboard
++ navigate most of the website using speech recognition software
++ listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+## How accessible this website is
+
+We know some parts of this website are not fully accessible because of [issues caused by our Technical Documentation Template](http://tdt-documentation.london.cloudapps.digital/accessibility/#using-the-technical-documentation-template-for-your-own-documentation).
```

Comment:
[Broken link](http://tdt-documentation.london.cloudapps.digital/accessibility/#using-the-technical-documentation-template-for-your-own-documentation)?

(Also, should be `https://`)

Reply:
Yes, sorry! This will be the correct URL but the statement isn't published yet. Forgot to add a note in the summary.

Reply:
Didn't spot the `https` though - thanks!